src/appkit.cs

```diff
@@ -6973,6 +6991,12 @@ partial interface NSFont : NSSecureCoding, NSCopying {
 		[Mac (11,0)]
 		[Export ("fontWithSize:")]
 		NSFont GetFont (nfloat fontSize);
+
+		[Advice ("Use constants from 'NSFontWeight' and 'NSFontWidth'.")]
+		[Mac (13,0)]
+		[Static]
+		[Export ("systemFontOfSize:weight:width:")]
+		NSFont GetSystemFontOfSize (nfloat fontSize, double weight, double width);
```

Comment:
Fonts need some custom treatment (look at the UIKit PR for the same selector): 

https://github.com/xamarin/xamarin-macios/pull/15694/files#diff-97d65d3ebcf51cbd79386a15db035a4a338cc9bf0fdcb2cd653d6b015ecd1770R6226